libdirections-offboard/src/main/java/com/mapbox/navigation/route/offboard/router/NavigationRouteCallback.kt

```diff
@@ -0,0 +1 @@
+
```

Comment:
Looks like rebase rubbish. I think we can remove it. The same for some other files in `offboard` module, please review it all